assets/src/models/searchQuery.ts

```diff
@@ -1,6 +1,9 @@
+export type SearchQueryType = "all" | "run" | "vehicle" | "operator"
```

Comment:
**praise:** hooray for better types!

Reply:
Yeah, I had to add `| string` to it's definition within the state because otherwise every usage required a `"all" as SearchQueryType`.